UNRELEASED.md

```diff
@@ -10,6 +10,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Added `download` prop to `Button` and `UnstyledLink` components to let you set the download attribute ([#1027](https://github.com/Shopify/polaris-react/pull/1027))
```

Comment:
```suggestion
- Added `download` prop to `Button` and `UnstyledLink` components that enables the download attribute ([#1027](https://github.com/Shopify/polaris-react/pull/1027))
```

Reply:
"that enables" feels like it's missing a word. "... components that enables setting the download attribute" feels nicer to me

Reply:
I was mostly just trying to avoid the use of `you`, but I don't feel strongly about it.

Reply:
"that enables setting" instead of "that enables" still removes the usage of "you" :)